bootstrap

```diff
@@ -5,11 +5,7 @@ set -x
 rm -rf autom4te*.cache
 
 # build it all.
-aclocal -I macros $ACLOCAL_FLAGS || exit 1
-autoheader || exit 1
-libtoolize --force --copy
-automake --include-deps --add-missing --foreign --copy || exit 1
-autoconf || exit 1
+autoreconf -fi || exit 1
 
 # Original configure call was:
```

Comment:
Side note: I think we can remove this very legacy comment

Reply:
Removed.